freqtrade/configuration/configuration.py

```diff
@@ -3,17 +3,14 @@
 """
```

Comment:
I think we need to either rename the module, or this file.

This can create confusion (which "configuration" is intendet this time?) - and can produce very subtle bugs when things are imported, since it's a namespace collision ("configuration" is both a module, and a file).

also, think about importing:
`from freqtrade.configuration.configuration import Configuration` ...

Reply:
`freqtrade/exchange/exchange.py`...

Reply:
lol yes :D

Reply:
I'm not against your proposal, not at all, but cannot find other appropriate names for the configuration dir and/or configuration.py...

Reply:
well me neither (that's why i didn't propose one)

the best is "config" and "configuration" - but that's even more misleading since the config dict is called "config" ... 

so let's leave it as is :+1: